### ISSUE TITLE
Empty Start Element Option

### ISSUE THREAD

**Issue by JRJurman** [Tram-One/tram-one#93]
## Summary
It can be useful to have content in your main div when Tram-One fails to load (for Search Engines or when JS is turned off). It appears right now if there is content in the main div, we're unable to render anything inside.

Right now this can be mitigated by emptying the element before starting:
```javascript
document.querySelector('.main').innerHTML = ''
app.start('.main')
```

We should have an option either when building (like `new Tram({defaultRoute: '/'})`) or starting (`.start`) Tram-One to effectively do this for us.

**Comment by JRJurman** [comment]
Conversely we could have an option to pre-render the home page with a route... 
Or both?

**Comment by JRJurman** [comment]
I think there was another issue here, and it is definitely NOT required to clear out the target div before attaching... One this happens again, we can document it here and re-open the issue.
